bin/dep

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+set -eu
+
+if [ "$(uname -s)" = "Darwin" ]; then
+  os=darwin
+else
+  os=linux
+fi
+if [ "$(uname -m)" = "x86_64" ]; then
+  arch=386
+else 
+  arch=amd64
+fi
+
```

Comment:
Ooh, this wrapper script gives us an opportunity to address a common `dep` pitfall: it [doesn't work from a symlinked project root](https://github.com/golang/dep/issues/1146). We could allow users to run from a symlinked dir if we add `cd $(pwd -P)` somewhere near the top of this script.